frontend/static_home_page/css/styles.css

```diff
@@ -127,191 +157,325 @@ h3 {
+    }
+}
+
+.homeTitleBox .titleText {
+    margin-top: 8px;
+    margin-bottom: 56px;
 }
 
 .homeContent2Col {
     display: flex;
-    top: 531px;
-    left: 219px;
-    width: 1700px;
-    position: absolute;
+    flex-direction: column;
+    
+}
+
+@media screen and (min-width: 1100px) {
+ .homeContent2Col {
+    flex-direction: row;
+    gap: 20px;
+ }   
 }
 
 .homeContentBox {
-    position: relative;
-    padding : 0px 40px 20px 0px;
+    background-color: #FFFFFF;
+    margin-bottom: 24px;
 }
 
+
+
 .homeContentBoxHeader {
-    background: #1F3564 0% 0% no-repeat padding-box;
-    height: 50px;
-    text-align : center;
+    background: var(--bc-background-blue);
+    padding: var(--padding-4) var(--padding-16);
+}
+
+.homeContentBoxHeader h2 {
+    margin: 0;
 }
 
 .homeContentBoxContent {
-    position: relative;
-    top: 0;
     background: #FFFFFF 0% 0% no-repeat padding-box;
-    padding : 10px 20px 10px 20px;
+    padding : var(--padding-24) var(--padding-16);
 }
 
+.homeContentBoxContent p {
+    margin-top: 0;
+}
+
+
 .homeContentBoxContent2Col {
-    position: relative;
-    top: 0;
+    border-top : 1px solid var(--bc-border-grey);
     display: flex;
+    flex-direction: column;
+}
+
+.homeContentBoxContent2Col .homeBorderedContentCol:not(:last-child) {
+    border-bottom:  1px solid var(--bc-border-grey);
+}
+
+@media screen and (min-width: 576px) {
+    .homeContentBoxContent2Col {
+        flex-direction: row;
+    }
+
+    .homeContentBoxContent2Col .homeBorderedContentCol {
+        width: 50%;
+    }
+
+    .homeContentBoxContent2Col .homeBorderedContentCol:not(:last-child) {
+        border-bottom: none;
+        border-right: 1px solid var(--bc-border-grey);
+    } 
+}
+
+.homeBorderedContentCol {
+    padding: var(--padding-16) var(--padding-16) var(--padding-24);
+}
+
+.homeContentCol {
+   width: 100%;
 }
 
 .homeContentCol1 {
-    flex : 2;
-    margin-right : 20px;
+    width: 100%;
 }
 
+
 .homeContentCol2 {
-    flex : 1;
-    padding-right: 40px
+    width: 100%;
+    display: flex;
+    flex-direction: column;
 }
 
-.homeBorderedContentCol {
-    border : 1px solid var(--bc-border-grey);
-    position: relative;
-    flex : 1;
-    padding: 0px 10px 10px 10px;
+@media screen and (min-width: 1100px) {
+    .homeContentCol1 {
+        width: 75%;
+    }
+    .homeContentCol2 {
+        width: 25%;
+    }
+    
+    .homeContentCol2 {
+        flex-direction: row;
+        gap: 20px;
+    }
+
+    .homeContentCol2 > * {
+        width: 100%;
+    }  
+}
+
+@media screen and (min-width: 1100px) {
+    .homeContentCol2 {
+        flex-direction: column;
+    }
 }
 
 .btn {
-    padding: 20px;
+    box-sizing: border-box;
+    padding: 10px 32px;
     text-decoration: none;
     display: block;
     text-align: center;
     opacity: 1;
     border-radius: 4px;
     font-family: var(--unnamed-font-family-bc-sans);
+    border: 4px solid transparent;
+    width: fit-content;
+}
+
+.btnSm {
```

Comment:
For descriptive class names like these (types of buttons), you can use BEM conventions (ie. for this small button, use ".btn--sm", and for blue buttons you can use ".btn--blue")